### DISCUSSION TITLE
Removing the metadata/keyword on elements

### DISCUSSION THREAD

**Discussion by bipulsarma** [structurizr/lite#111] [Q&A] [answered]
Is there a way to remove the metadata or keyword on the elements in structurizr? For example when I build a diagram that includes a user can I not show the metadata/keword [person] in the visual?

**Comment by simonbrowndotje** [comment, marked as answer]
If you're using the DSL, add `metadata false` to your style definition; for example:

```
workspace {

    model {
        user = person "User" "A user of my software system."
        softwareSystem = softwareSystem "Software System" "My software system."

        user -> softwareSystem "Uses"
    }

    views {
        systemContext softwareSystem "SystemContext" {
            include *
            autoLayout
        }

        styles {
            element "Person" {
                shape person
                metadata false
            }
        }
    }
    
}
```

Related links:
 - https://docs.structurizr.com/ui/diagrams/notation#elements
 - https://docs.structurizr.com/dsl/language#element-style

**Reply by bipulsarma** [reply]
Thanks, that's what I was looking for.
